src/main/kotlin/com/faforever/userservice/config/DatabaseConfig.kt

```diff
@@ -0,0 +1,39 @@
+package com.faforever.userservice.config
+
+import io.r2dbc.spi.ConnectionFactory
+import org.slf4j.LoggerFactory
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.core.convert.converter.Converter
+import org.springframework.data.convert.CustomConversions
+import org.springframework.data.convert.ReadingConverter
+import org.springframework.data.r2dbc.convert.R2dbcCustomConversions
+import org.springframework.data.r2dbc.dialect.DialectResolver
+import java.time.LocalDateTime
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+
+@ReadingConverter
+class LocalDateTimeToOffsetDateTimeReadingConverter : Converter<LocalDateTime, OffsetDateTime> {
+    private val log = LoggerFactory.getLogger(LocalDateTimeToOffsetDateTimeReadingConverter::class.java)
+
+    override fun convert(source: LocalDateTime): OffsetDateTime {
+        log.debug("convert() called with: source = $source")
+        return source.atOffset(ZoneOffset.UTC)
+    }
+}
+
+@Configuration
+class DatabaseConfiguration {
+    @Bean
+    fun r2dbcCustomConversions(connectionFactory: ConnectionFactory): R2dbcCustomConversions {
+        val dialect = DialectResolver.getDialect(connectionFactory)
+        val converters: MutableList<Any> = ArrayList(dialect.converters)
+        converters.addAll(R2dbcCustomConversions.STORE_CONVERTERS)
```

Comment:
the sound version in Kotlin is `listOf()` or in case it *must* be a mutable list then `mutableListOf(...)`
If needed also `listOf<Any>` to get rid of the variable type

Reply:
Over and out